x-pack/platform/plugins/private/index_lifecycle_management/kibana.jsonc

```diff
@@ -3,7 +3,7 @@
   "id": "@kbn/index-lifecycle-management-plugin",
   "owner": "@elastic/kibana-management",
   "group": "platform",
-  "visibility": "private",
+  "visibility": "shared",
```

Comment:
I think this change means this should be moved `x-pack/platform/plugins/private/index_lifecycle_management` => `x-pack/platform/plugins/shared/index_lifecycle_management`